.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Should `.idea` directory be ignored?

Reply:
Oops, good catch—included it on accident. Removed in https://github.com/WebAssembly/wasi-messaging/pull/24/commits/f3658470dd6cc0dc07fd4b11f2e3bd474718f85a 👍